projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/_overrides.scss

```diff
@@ -134,3 +134,5 @@
 .admin-color-contrast #adminmenuback {
 	outline: 1px solid $nav-unification-sidebar-border;
 }
+
+@import "../_upsell-banner";
```

Comment:
Any reason for importing this file here as opposed to do it in the `_admin.scss` file like we do for other files (e.g. `gutenberg` or `inline-help`).

Reply:
I was trying something, can't recall if there was a reason. It works perfectly fine in `_admin.scss` efd046f.

Good catch!

Reply:
Actually there was a reason after all... 
`@import "../admin";` (which loads `@import 'overrides';`) is only loaded for Calypso Color schemes.
`@import "../_core-overrides";` is loaded for WP Admin Color schemes.

I finally followed the `inline-help` path, adding `@import "../_upsell-banner";` both to `_admin.scss` and each Calypso Color scheme. It took me a while to remember this... 

e008077